src/lib/containers/UserCardList.tsx

```diff
@@ -53,8 +54,10 @@ export default class UserCardList extends React.Component<
     }
   }
 
-  update(list: string[]) {
-    getUserProfileWithProfilePicAttached(list.filter(el => el))
+  update(list: (string | null)[]) {
+    getUserProfileWithProfilePicAttached(
+      list.filter((el): el is string => !!el),
```

Comment:
For this call, filter the null values (they obviously won't have profiles we can fetch)